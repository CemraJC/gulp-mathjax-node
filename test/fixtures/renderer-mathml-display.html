<!DOCTYPE html>
<html>
<head>
    <title>MathML Rendering</title>
</head>
<body>

<script type="math/tex; mode=display">\int_0^6 \frac{5x}{3} dx</script>
$$
    \int_0^6 \frac{5ex}{3} dex
$$

</body>
</html>