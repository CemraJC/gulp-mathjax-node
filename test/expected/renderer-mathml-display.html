<!DOCTYPE html>
<html>
<head>
    <title>MathML Rendering</title>
</head>
<body>

<math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><msubsup><mo stretchy="false">∫</mo><mn>0</mn><mn>6</mn></msubsup><mfrac><mrow><mn>5</mn><mi>x</mi></mrow><mn>3</mn></mfrac><mi>d</mi><mi>x</mi></math>
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><msubsup><mo stretchy="false">∫</mo><mn>0</mn><mn>6</mn></msubsup><mfrac><mrow><mn>5</mn><mi>e</mi><mi>x</mi></mrow><mn>3</mn></mfrac><mi>d</mi><mi>e</mi><mi>x</mi></math>

</body>
</html>